Editor/AGXUnityEditor/Menus/TopMenu.cs

```diff
@@ -385,6 +386,20 @@ public static GameObject Simulation()
       return Selection.activeGameObject = GetOrCreateUniqueGameObject<Simulation>()?.gameObject;
     }
 
+    [MenuItem( "AGXUnity/Plot", priority = 66 )]
+    public static GameObject Plot()
+    {
+      var PlotObject = new GameObject("PlotObject");
+      PlotObject.AddComponent<AGXUnity.Utils.Plot>();
+      PlotObject.AddComponent<AGXUnity.Utils.DataSeries>();
+      PlotObject.AddComponent<AGXUnity.Utils.DataSeries>();
+      var plotAssetPath = AGXUnityEditor.IO.Utils.AGXUnityResourceDirectory + "/Plot/Plot.Asset";
+      if (!System.IO.File.Exists("Assets/Plot.Asset"))
```

Comment:
Should we create a new asset for each plot object we create? If so we can do something similar to what AssetFactory does to create a copy with a unique name.